tests/monitoring_test.go

```diff
@@ -81,6 +84,58 @@ func countAlerts(alerts []alert) int {
 	return len(alerts)
 }
 
+var _ = Describe("Exporters", func() {
+	var c kubernetes.Interface
+
+	BeforeEach(func() {
+		c = kubernetes.NewForConfigOrDie(clusterConfigOrDie())
+	})
+
+	DescribeTable("Exported timeseries", func(selector string, match types.GomegaMatcher) {
+		params := map[string]string{
+			"match[]": selector,
+			"start":   fmt.Sprintf("%d", time.Now().Add(-20*time.Minute).Unix()),
+		}
+		resultRaw, err := c.CoreV1().
+			Services("kubeprod").
+			ProxyGet("http", "prometheus", "9090", "api/v1/series", params).
+			DoRaw()
+		Expect(err).NotTo(HaveOccurred())
+
+		resp := promResponse{}
+		json.Unmarshal(resultRaw, &resp)
+		var series []map[string]string
+		json.Unmarshal(resp.Data, &series)
+
+		fmt.Fprintf(GinkgoWriter, "%s found %d timeseries:\n", selector, len(series))
+		for i, s := range series {
+			if i >= 10 {
+				fmt.Fprintf(GinkgoWriter, "(truncated ...)\n")
+				break
+			}
+			fmt.Fprintf(GinkgoWriter, "%s{", s["__name__"])
+			for k, v := range s {
+				if k != "__name__" {
+					fmt.Fprintf(GinkgoWriter, "%s=%q,", k, v)
+				}
+			}
+			fmt.Fprintf(GinkgoWriter, "}\n")
+		}
+
+		Expect(series).To(match)
```

Comment:
Prometheus may not be ready when we launch the tests in the CI. We need to have some retries in this test.

Reply:
The `kubectl rollout` command can only report rollout of Deployments. Prometheus and Elasticsearch use Statefulsets. I couldn't find a way to make it work for statefulsets.

Reply:
When I tried using `kubectl rollout` with a sts, I remember it would always report that it was not ready despite being actually rolled out.

Reply:
Added https://github.com/bitnami/kube-prod-runtime/pull/257 to support STS